docs/src/legacy/child_device_config_management_agent.md

```diff
@@ -225,7 +225,6 @@ a "failed" status update must be sent instead, to the same topic as follows:
 
 ## References
 
-* Configuration Management [documentation](config_management_plugin.md)
 * Reference implementation of a [child device agent](https://github.com/thin-edge/thin-edge.io_examples/tree/main/child-device-agent) written in Python to demonstrate the contract described in this document.
 
 
```

Comment:
We can get rid of this section as we'll be updating the reference child device agent to use the new v1 syntax, so it won't match the contract described on this page anyway.

Reply:
removed